src/app/work-item/work-item-list/work-item-detail/work-item-link/work-item-link.service.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable } from '@angular/core';
+import { Headers, Http } from '@angular/http';
+
+import 'rxjs/add/operator/toPromise';
+
+import { AuthenticationService } from '../../../../auth/authentication.service';
+import { Logger } from '../../../../shared/logger.service';
+import { LinkType } from '../../../../models/link-type';
+import { Link } from '../../../../models/link';
+
+
+@Injectable()
+export class WorkItemLinkService {
+  private headers = new Headers({'Content-Type': 'application/vnd.api+json'});
+  // Fetching link category of system for now (To be refactored after full implementation)
+  private linkCatgoryUrl = process.env.API_URL + 'linkcategories/1';
```

Comment:
The URL needs to be updated from `linkcategories/` to `workitemlinkcategories/`.